com.bitwarden.desktop.yaml

```diff
@@ -17,65 +20,181 @@ finish-args:
+      - install -Dm644 com.bitwarden.desktop.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 com.bitwarden.desktop.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 755 bitwarden.sh ${FLATPAK_DEST}/bin/bitwarden
 
       # To allow separate locales
       # https://searchfox.org/mozilla-central/rev/8a4f55bc09ffc5c25dcb4586c51ae4a9fee77b4c/taskcluster/docker/firefox-flatpak/runme.sh#131-133
       - |
-        for lang in ${FLATPAK_DEST}/Bitwarden/locales/*.pak
+        for lang in ${FLATPAK_DEST}/bitwarden-desktop/locales/*.pak
         do
           locale="$(basename -s .pak $lang)"
           install -Dm644 -t "${FLATPAK_DEST}/share/runtime/locale/${locale%%-*}/" "$lang"
-          ln -sf "${FLATPAK_DEST}/share/runtime/locale/${locale%%-*}/$(basename $lang)" "${FLATPAK_DEST}/Bitwarden/locales/$(basename $lang)"
+          ln -sf "${FLATPAK_DEST}/share/runtime/locale/${locale%%-*}/$(basename $lang)" "${FLATPAK_DEST}/bitwarden-desktop/locales/$(basename $lang)"
         done
     sources:
-      - type: file
-        only-arches:
-          - x86_64
-        url: https://github.com/bitwarden/clients/releases/download/desktop-v2023.1.1/Bitwarden-2023.1.1-amd64.deb
-        sha256: 6cbdf26c4ae96398de0a2c45f2ab54fc3437e7153ee372bd697ade1eca0217b4
-        x-checker-data:
-          type: anitya
-          project-id: 179174
-          url-template: https://github.com/bitwarden/clients/releases/download/desktop-v$version/Bitwarden-$version-amd64.deb
-          is-main-source: true
+      - type: git
+        url: https://github.com/bitwarden/clients.git
+        commit: ff084f6fce39ff535bfbc69f076516e869395d0d
       - type: script
         dest-filename: bitwarden.sh
         commands:
           - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
-          - exec zypak-wrapper /app/Bitwarden/bitwarden "$@"
+          - exec zypak-wrapper /app/bitwarden-desktop/bitwarden "$@"
       - type: file
         path: com.bitwarden.desktop.metainfo.xml
-
-  - name: bitwarden-cli
-    buildsystem: simple
-    build-commands:
-      - unzip bw-linux*.zip
-      - install -D -m 755 -t "${FLATPAK_DEST}/bin" bw
-    sources:
       - type: file
-        only-arches:
-          - x86_64
-        url: https://github.com/bitwarden/clients/releases/download/cli-v2023.1.0/bw-linux-2023.1.0.zip
-        sha256: 523684cc0b4f048c923525c2e23047b85ddb72341ea25c2fb3b140cbd2e37e8d
-        x-checker-data:
-          type: anitya
-          project-id: 229011
-          url-template: https://github.com/bitwarden/clients/releases/download/cli-v$version/bw-linux-$version.zip
+        path: com.bitwarden.desktop.desktop
+
+      # flatpak-node-generator --electron-node-headers npm package-lock.json -o node-sources.json
+      - node-sources.json
+      # flatpak-cargo-generator.py Cargo.lock -o cargo-sources.json
+      - cargo-sources.json
+
+  # - name: bitwarden-cli
```

Comment:
you can remove all those commented lines as they are unecessary.

Reply:
Removed. CLI is somewhat hard to build in flatpak at the moment, but looking to re-introduce at some point (maybe as a separate package?).